src/client.rs

```diff
@@ -40,27 +40,32 @@ impl Client {
     ) -> Self {
         Client {
             client,
+            field_manager: field_manager.clone(),
             post_params: PostParams {
                 field_manager: field_manager.clone(),
                 ..PostParams::default()
             },
             patch_params: PatchParams {
-                field_manager: field_manager.clone(),
-                ..PatchParams::default()
-            },
-            // TODO: According to https://kubernetes.io/docs/reference/using-api/server-side-apply/#using-server-side-apply-in-a-controller we should always force conflicts in controllers.
-            //  we currently consider this a workaround until we can properly implement patching
-            //  (see https://github.com/stackabletech/operator-rs/issues/113)
-            force_patch_params: PatchParams {
                 field_manager,
-                force: true,
                 ..PatchParams::default()
             },
             delete_params: DeleteParams::default(),
             default_namespace,
         }
     }
 
+    /// Server-side apply requires a `field_manager` that uniquely identifies a single usage site,
+    /// since it will revert changes that are owned by the `field_manager` but not part of the Apply request.
+    fn apply_patch_params(&self, context: impl Display) -> PatchParams {
+        let mut params = self.patch_params.clone();
+        // TODO: According to https://kubernetes.io/docs/reference/using-api/server-side-apply/#using-server-side-apply-in-a-controller we should always force conflicts in controllers.
```

Comment:
Do we still need this todo, seems like the next line forces conflicts, which this comment is about unless I misread it?

Reply:
Oh right, that shouldn't be a TODO anymore. Think there's still value in keeping it there as a reference though.